packages/atoms/src/ListItem/SelectableListItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { TouchableOpacity, TouchableOpacityProps, View } from 'react-native';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ListItemIcon } from './ListItemIcon';
+import { ListItemText } from './ListItemText';
+import { Radio, RadioProps } from '../Radio';
+import { BaseListItem } from './listItemTypes';
+import { Box } from '../Box';
+
+type SelectableListItemProps = RadioProps &
+    Omit<BaseListItem, 'onPress'> &
+    Omit<TouchableOpacityProps, 'style' | 'onPress'>;
+
+const radioStyle = prepareNativeStyle(() => ({
```

Comment:
nit: radioContainerStyle

Reply:
Refactored to Box component in https://github.com/trezor/trezor-suite/pull/5482/commits/81e57f45bd044390d8b19895426a9516cff9256b